### DISCUSSION TITLE
Forcing local scope in quarto

### DISCUSSION THREAD

**Discussion by danieltomasz** [quarto-dev/quarto-cli#6441] [Q&A] [answered]
### Description

How I can force local scope in quarto looking for `_quarto.yml`?
Often I have `_quarto.yml` several folders higher in hierarchy and their are picked If I do not another  `_quarto.yml` in the subfolder, but often I would like to turn do it per file basis.

**Comment by mcanouil** [comment, marked as answer]
Adding `_quarto.yml` is the way to indicate to Quarto the scope, especially if you n'est Quarto projects.
It seems pretty straightforward and explicit, what else would you want to have?

**Reply by danieltomasz** [reply]
Thanks!
 Sometimes  when I am testing something I would like to   compile file while  ignoring `_quarto.yml` (otherwise I need to create many redudant (`_quarto.yaml` or  create another yaml file with separate profie) but I could manage with the current system.

I work with Obsidian with many nested folders, but often I have many nested VSCode projects within  those subfolders;
When I have too many duplicated `_quarto.yml` it is hard to me propagate changes/updates to these setting (quite often I tweak tex preamble and change tex settings); 

Last question - how  I can force quarto to produce pdf in current folder?
Within `_quarto.yml` within root    I have often

```
project:
  type: book
  output-dir: _output
```

This is most useful for me in general but sometimes I have  file nested many levels  and need to work from not root buts some subfolder.
When I run quarto I produce pdf nested many  folders with `_output` subfolder, but not in my current folder which is inconvenient ( I would like to use all the other settings from  `_quarto.yml` )

**Reply by mcanouil** [reply]
I believe you need to define your Quarto profiles for your use case and possibly make them mutually exclusive and set a default one as explained in <https://quarto.org/docs/projects/profiles.html>.

To note, nested projects is not usually and in general a good practice as it makes everything more complicated.
